src/api/stats.c

```diff
@@ -196,23 +196,6 @@ int api_stats_top_domains(struct ftl_conn *api)
 	// Sort temporary array
 	qsort(temparray, added_domains, sizeof(int[2]), cmpdesc);
 
-	// Get filter
-	const char* log_show = read_setupVarsconf("API_QUERY_LOG_SHOW");
-	bool showpermitted = true, showblocked = true;
```

Comment:
I wonder if we should re-add the simple show blocked/permitted checkbox options. I know we have regex client/domain filtering now, but the checkboxes might be nice for novice users.

Reply:
Changes to this file are undone after merging `development-v6`